site/en/external/registry.md

```diff
@@ -1,103 +1,171 @@
+[_index registries_](#index_registry) — local directories or static HTTP servers
 following a specific format.
 
 ## Index registry
 
 An index registry is a local directory or a static HTTP server containing
 information about a list of modules — including their homepage, maintainers, the
 `MODULE.bazel` file of each version, and how to fetch the source of each
-version. Notably, it does *not* need to serve the source archives itself.
+version. Notably, it does _not_ need to serve the source archives itself.
 
 An index registry must follow the format below:
 
-*   `/bazel_registry.json`: A JSON file containing metadata for the registry
-    like:
-    *   `mirrors`: specifying the list of mirrors to use for source archives.
-        The mirrored URL is a concatenation of the mirror itself, and the
-        source URL of the module specified by its `source.json` file sans the
-        protocol. For example, if a module's source URL is
-        `https://foo.com/bar/baz`, and `mirrors` contains
-        `["https://mirror1.com/", "https://example.com/mirror2/"]`, then the
-        URLs Bazel will try in order are `https://mirror1.com/foo.com/bar/baz`,
-        `https://example.com/mirror2/foo.com/bar/baz`, and finally the original
-        source URL itself `https://foo.com/bar/baz`.
-    *   `module_base_path`: specifying the base path for modules with
-        `local_repository` type in the `source.json` file
-*   `/modules`: A directory containing a subdirectory for each module in this
-    registry
-*   `/modules/$MODULE`: A directory containing a subdirectory for each version
-    of this module, as well as:
-    *   `metadata.json`: A JSON file containing information about the module,
-        with the following fields:
-        *   `homepage`: The URL of the project's homepage
-        *   `maintainers`: A list of JSON objects, each of which corresponds to
-            the information of a maintainer of the module *in the registry*.
-            Note that this is not necessarily the same as the *authors* of the
-            project
-        *   `versions`: A list of all the versions of this module to be found in
-            this registry
-        *   `yanked_versions`: A map of [*yanked*
-            versions](/external/module#yanked_versions) of this module. The keys
-            should be versions to yank and the values should be descriptions of
-            why the version is yanked, ideally containing a link to more
-            information
-*   `/modules/$MODULE/$VERSION`: A directory containing the following files:
-    *   `MODULE.bazel`: The `MODULE.bazel` file of this module version
-    *   `source.json`: A JSON file containing information on how to fetch the
-        source of this module version
-        *   The default type is "archive", representing an `http_archive` repo,
-            with the following fields:
-            *   `url`: The URL of the source archive
```

Comment:
can you revert the changes on this page and only add the following part to this line?

```
     * `mirror_urls`: A list of string, the mirror URLs of the source archive.
    The URLs are tried in order after `url` as backups.
```